package.json

```diff
@@ -5,53 +5,51 @@
   "main": "index.js",
   "scripts": {
     "lint": "eslint",
-    "test": "mocha  --require babel-register tests/*.js --exit",
+    "test": "npm run create_tables && NODE_ENV=test nyc --reporter=text mocha --require babel-register --require babel-polyfill ./tests/ --exit && npm run drop_tables",
     "coveralls": "npm test && nyc report --reporter=text-lcov | coveralls",
     "coverage": "npm test && nyc report --reporter=text-lcov | coveralls",
-    "start": "babel-node index.js",
-    "dev": "nodemon --exec babel-node index.js"
+    "drop_tables": "NODE_ENV=test sequelize db:migrate:undo:all",
+    "create_tables": "NODE_ENV=test sequelize db:migrate",
+    "start": "node --exec babel-node ./src/index.js",
+    "dev": "npm run create_tables && nodemon --exec babel-node ./src/index.js"
```

Comment:
you can't create tables every time you make a change to the code. They should be created once.
Remove create tables

The coverage and coveralls commands are the same. Remove one of them.